### PR TITLE
refactor: tweak graph const. measurement

### PR DESCRIPTION
**Pull request by stephanwlee**
Polymer's property setter invokes binding and causes ripple of changes.
Previously, with new Hierarchy() measurement, we were measuring time it
takes to propagate `renderHierarchy` changes which internally redoes
layout and rendering. As such, `new Hierarchy` was measuring,
"tf-graph-scene (layout)" and "tf-graph-scene (build scene)".

### CODE REVIEW

File: tensorboard/plugins/graph/tf_graph/tf-graph.ts [hunk 1]
```diff
@@ -298,12 +302,12 @@ class TfGraph extends LegacyElementMixin(PolymerElement) {
...
-        this.async(function () {
-          this.fire('rendered');
-        });
-      }.bind(this)
+
+        return renderGraph;
+      }
     );
+
+    anyThis._setRenderHierarchy(renderGraph);
```
Comment (psybuzz) [inline tensorboard/plugins/graph/tf_graph/tf-graph.ts:310]: Before, `_buildRenderHierarchy` was a no-op when `graphHierarchy.root.type !== tf_graph.NodeType.META`.  Now, we will end up calling `anyThis._setRenderHierarchy(undefined)`.  Is this change to logic intentional and safe?

Or does it make sense to pull our the non-meta-root-node check to the very top of `_buildRenderHierarchy` and do an early return?

File: tensorboard/plugins/graph/tf_graph/tf-graph.ts [hunk 2]
```diff
@@ -298,12 +302,12 @@ class TfGraph extends LegacyElementMixin(PolymerElement) {
             };
           }),
         });
-        this._setRenderHierarchy(renderGraph);
-        this.async(function () {
-          this.fire('rendered');
-        });
```
Comment (psybuzz) [inline tensorboard/plugins/graph/tf_graph/tf-graph.ts:304]: I found no places where we listen for a 'rendered' event or have `on-rendered` in a template, so I'm assuming this was just unused.

**Review by psybuzz — APPROVED** [review]
lgtm % comment about non-root-meta-node.

File: tensorboard/plugins/graph/tf_graph/tf-graph.ts (hunk 2 above)
Comment (stephanwlee) [reply tensorboard/plugins/graph/tf_graph/tf-graph.ts:304]: I think so. I cannot find any place where we use it.

**Pull request merged**